ebmlite/core.py

```diff
@@ -42,14 +42,14 @@
 __all__ = ['BinaryElement', 'DateElement', 'Document', 'Element',
            'FloatElement', 'IntegerElement', 'MasterElement', 'Schema',
            'StringElement', 'UIntegerElement', 'UnicodeElement',
-           'UnknownElement', 'VoidElement', 'loadSchema']
+           'UnknownElement', 'VoidElement', 'loadSchema', 'parseSchema']
```

Comment:
this looks harmless but I'm not sure what it does; anything notable that I should know about this? was it just left out by accident before?

Reply:
Yeah, I left it out of `__all__` when I added the function on 2021-11-08.